lighthouse-core/audits/audit.js

```diff
@@ -102,9 +101,9 @@ class Audit {
   }
 
   /**
-   * @param {!Audit} audit
-   * @param {!AuditResult} result
-   * @return {{score: number, scoreDisplayMode: string}}
+   * @param {typeof Audit} audit
+   * @param {LH.AuditResult} result
+   * @return {{score: number, scoreDisplayMode: Audit.ScoringModeValues}}
```

Comment:
`Audit.ScoringModeValues` vs `@typedef {Object} Audit.ScoringModes` below?

Reply:
`ScoringModeValues` is one of the string values of the `Audit.ScoringModes` object (so just `'numeric' | 'binary'`). We could probably fix this up better, maybe if we pull most of the audit stuff into here and/or an `audit.d.ts` file